templates/dashboard/base_modal.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+
+<form
+    role="form" method="post" action="{% block action_url %}#!{% endblock %}"
+    class="{% block form_class %}form-async{% endblock %}" novalidate>
```

Comment:
Here we define class `form-async`, but in some modal templates in this PR we override it with an empty block and in some other, we don't. Why is that? Is that inconsistency in how our modals work?

Reply:
The reason is that when we perform remove actions we shouldn't make ajax request and go back to model details page (like form-async class treats such modals), but rather simply process form and redirect to address returned from server (usually a list).

Reply:
Makes sense but it could be documented because either we forget about it or someone else creating a new template will not know how to use it. A comment in this template would be useful.